config/index.js

```diff
@@ -112,4 +112,5 @@ module.exports = {
       sortOrder: 2,
     },
   },
+  MOVES_BATCH_SIZE: process.env.MOVES_BATCH_SIZE || 40,
```

Comment:
Could we add this to the readme too please :)

I wonder if `LOCATIONS_BATCH_SIZE` might also be a better a name as its referring to when we have lots of locations rather than moves